src/main/protobuf/ssl-refbox/Referee.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto2";
+option java_package = "org.robocup.ssl";
```

Comment:
I think a reference to the source of these files is useful. maybe even add a version/commit hash.